README.md

```diff
@@ -101,6 +101,20 @@ var tsml_react_config = {
 };
 ```
 
+You can optionally also specify a `tiles_dark` attribute which will be used when `prefers-color-scheme: dark`.
+
+```js
+var tsml_react_config = {
+  map: {
+    tiles_dark: {
+      attribution:
+        '&copy; <a href="https://www.stadiamaps.com/" target="_blank">Stadia Maps</a> &copy; <a href="https://openmaptiles.org/" target="_blank">OpenMapTiles</a> &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+      url: 'https://tiles.stadiamaps.com/tiles/alidade_smooth_dark/{z}/{x}/{y}{r}.png',
```

Comment:
I think stadiamaps requires a api key, like it works fine on 127.0.0.1 or localhost but in production you need a key. so to use darkmode you would have to do `url: 'https://tiles.stadiamaps.com/tiles/alidade_smooth_dark/{z}/{x}/{y}{r}.png?api_key=db30b16c-caba-blah',`

Reply:
see https://gist.github.com/BrendonKoz/b1df234fe3ee388b402cd8e98f7eedbd I did see a response on this fun thread from OSM team that they are working on actual darkmode vectors https://en.osm.town/@osm_tech/113482687301656785

Reply:
thanks that's a good callout - this is the dark mode value that i added to the aasanjose story (i didn't need an API key but i did need to set up authorized domains). i just added this as an example for the documentation, but perhaps i should either make this obviously fictitious or pick another one that works out of the box?

Reply:
I couldn't find one that didn't need a key, I think either make fictions or just add note about possibly needing keys for different map providers

Reply:
this totally works

```js
map: {
    tiles: {
        attribution: `Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>`,
        url: 'https://api.mapbox.com/styles/v1/mapbox/streets-v11/tiles/{z}/{x}/{y}?access_token=pk.eyJ1Ij_MY_KEY_HERE',
    },
    tiles_dark: {
        attribution: `Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>`,
        url: 'https://api.mapbox.com/styles/v1/mapbox/dark-v10/tiles/{z}/{x}/{y}?access_token=pk.eyJ1Ij_MY_KEY_HERE',
    },
}
```

Reply:
so if you wanted to you could still support mapbox with the smallest of code changes (just pass in key to tiles)